src/task_queue/database_tasks/media_library_cleanup.py

```diff
@@ -0,0 +1,59 @@
+import datetime
+from django.http import HttpResponse
+from _main_.utils.emailer.send_email import send_massenergize_email_with_attachments
+from api.store.common import (
+    find_duplicate_items,
+    get_admins_of_communities,
+    get_duplicate_count,
+    remove_duplicates_and_attach_relations,
+    summarize_duplicates_into_csv,
+)
+from api.utils.constants import MEDIA_LIBRARY_CLEANUP_TEMPLATE
+from database.models import FeatureFlag
+
+
+REMOVE_DUPLICATE_IMAGE_FLAG_KEY = "remove-duplicate-images-feature-flag"
+
+
+def remove_duplicate_images():
+    """
+    This checks all media on the platform and removes all duplicates.
+    Its based on the "Remove Duplicate Images" feature flag. For communities that are subscribed
+    to the flag, duplicates will be removed from their libraries.
+    """
+    try: 
+        flag = FeatureFlag.objects.filter(key=REMOVE_DUPLICATE_IMAGE_FLAG_KEY).first()
+
+        for community in flag.communities.all():
+            ids = [community.id]
+            grouped_dupes = find_duplicate_items(False, community_ids=ids)
+            num_of_dupes_in_all = get_duplicate_count(grouped_dupes)
+            csv_file = summarize_duplicates_into_csv(grouped_dupes)
+            admins = get_admins_of_communities(ids)
+
+            for hash_value in grouped_dupes.keys(): 
+                remove_duplicates_and_attach_relations(hash_value)
+
+            for admin in admins:
```

Comment:
just send the csv to the task owner.  community admins don't need to know about this

Reply:
Noted!

Reply:
Added this, but now you will have to name the task "Media Library Cleanup Routine" when you create it. Or make the code match what name you choose if you do change it.